tools/triage_status_report.py

```diff
@@ -22,6 +22,11 @@ class IssueData:
     url: str
     features: List[str]
 
+    def __post_init__(self):
+        for field, value in self.__dict__.items():
+            if value is None:
+                raise ValueError(f"Field '{field}' cannot be None")
```

Comment:
Can we maybe collect all those exceptions and show them as part of the Slack message, instead of giving up completely? Something like 

"Here are the triage tickets:

And here are the triage tickets that could not be parsed: 

"